hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -475,13 +485,13 @@ else:
     max_intrwidth = max([len(x["name"]) for x
         in m["interrupt_list"]])
 %>\
-  % if "parameter" in m:
+  % if len(m["param_list"]) != 0:
```

Comment:
This would probably be more "Pythonic" as just `if m['param_list']:` (because the empty list is falsy in Python)

Reply:
Done.